highway is untagged and have no other consumers. Keep it private to
libjxl package for now.

--- lib/jxl.cmake.orig	2021-10-04 18:02:44 UTC
+++ lib/jxl.cmake
@@ -557,7 +557,7 @@ endif()  # TARGET_SUPPORTS_SHARED_LIBS AND NOT JPEGXL_
 
 # Add a pkg-config file for libjxl.
 set(JPEGXL_LIBRARY_REQUIRES
-    "libhwy libbrotlicommon libbrotlienc libbrotlidec")
+    "libbrotlicommon libbrotlienc libbrotlidec")
 if(NOT JPEGXL_ENABLE_SKCMS)
   set(JPEGXL_LIBRARY_REQUIRES "${JPEGXL_LIBRARY_REQUIRES} lcms2")
 endif()
--- lib/jxl/libjxl.pc.in.orig	2021-10-04 18:02:44 UTC
+++ lib/jxl/libjxl.pc.in
@@ -8,5 +8,5 @@ Description: Loads and saves JPEG XL files
 Version: @JPEGXL_LIBRARY_VERSION@
 Requires.private: @JPEGXL_LIBRARY_REQUIRES@
 Libs: -L${libdir} -ljxl
-Libs.private: -lm
+Libs.private: -lhwy -lm
 Cflags: -I${includedir}
--- third_party/CMakeLists.txt.orig	2020-11-14 00:52:03 UTC
+++ third_party/CMakeLists.txt
@@ -83,7 +83,7 @@ endif()  # BUILD_TESTING
 # Highway
 if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/highway/CMakeLists.txt" AND
     NOT JPEGXL_FORCE_SYSTEM_HWY)
-  add_subdirectory(highway)
+  add_subdirectory(highway EXCLUDE_FROM_ALL)
   configure_file("${CMAKE_CURRENT_SOURCE_DIR}/highway/LICENSE"
                  ${PROJECT_BINARY_DIR}/LICENSE.highway COPYONLY)
 else()
--- third_party/highway/CMakeLists.txt.orig	2020-12-02 19:39:56 UTC
+++ third_party/highway/CMakeLists.txt
@@ -230,7 +230,7 @@ set_target_properties(hwy_benchmark
 
 include(CTest)
 
-if(BUILD_TESTING)
+if(HWY_BUILD_TESTING)
 enable_testing()
 include(GoogleTest)
 
